tests/tasks/TestIpfsUpload.test_ipfs_upload_successful.approved.txt

```diff
@@ -1,9 +1,5 @@
 
+⠋ Uploading[KHTTP Request: POST https://api.pinata.cloud/pinning/pinFileToIPFS "HTTP/1.1 200 OK"
 
-
-⠋ UploadingDEBUG: HTTP Request: POST https://api.pinata.cloud/pinning/pinFileToIPFS "HTTP/1.1 200 OK"
-
-            
-✅  Uploading
-File uploaded successfully!
-CID: test
+ File uploaded successfully!
+ CID: test
```

Comment:
Should there be a snapshot of successfull completion of uploading?

Reply:
That's what this is. It makes sense to me that the success scenario is tested.
Not sure I fully understand the comment.

Reply:
all good, must have confused the diff, resolving